devices/ePaper/Graphics.cs

```diff
@@ -0,0 +1,388 @@
+﻿// Copyright (c) 2022 The nanoFramework project contributors
+// See LICENSE file in the project root for full license information.
+
+using System;
+
+using Iot.Device.ePaper.Buffers;
+using Iot.Device.ePaper.Drivers;
+using Iot.Device.ePaper.Fonts;
+using Iot.Device.ePaper.Primitives;
+
+namespace Iot.Device.ePaper
+{
+    /// <summary>
+    /// A graphics class for ePaper displays with basic graphic APIs support.
+    /// </summary>
+    public sealed class Graphics : IDisposable
+    {
+        private bool disposedValue;
```

Comment:
our namig convetion says that privat fields should start with underscore _
So I really think the linter is not properly activated